apps/console/src/features/identity-providers/components/wizards/enterprise-idp-create-wizard.tsx

```diff
@@ -388,6 +388,12 @@ export const EnterpriseIDPCreateWizard: FC<EnterpriseIDPCreateWizardProps> = (
                 errors.name = t("console:develop.features.authenticationProvider." +
                     "forms.generalDetails.name.validations.duplicate");
             }
+            // We have the inputType "identifier" but however it has no
+            // effect because validate function is overridden by this function.
+            // So, we need re-check this manually.
+            if (!FormValidation.identifier(values.name)) {
+                errors.name = "Please enter a valid name";
```

Comment:
Ideally, this should be localized, right?

Reply:
Addressed with 8c6e4d1